clash-lib/src/Clash/Netlist/BlackBox/Types.hs

```diff
@@ -121,7 +121,7 @@ data Element
   -- ^ Like Arg, but input hole must be a literal
   | Name !Int
   -- ^ Name hole
-  | Var [Element] !Int
+  | ToVar [Element] !Int
```

Comment:
Why was this renamed?

Reply:
It's difficult to see where this template tag is used right now due to the naming collision with the other `Var`. My plan was to make it a separate PR. (Any commit not related to temporal logic should go in a separate PR.)

Reply:
Okay, marking as resolved